ask-sdk-controls/README.md

```diff
@@ -9,30 +9,4 @@
 
 The ASK SDK Controls framework builds on the [ASK SDK for Node.js](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs), offering a scalable solution for creating large, multi-turn skills in code with reusable components called *controls*.
 
-## Documentation
-
-- [About the Controls Framework](https://alexa.design/controls)
-- [User Guide](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/tree/2.0.x/ask-sdk-controls/doc/userGuide.md)
-- [API Reference](http://ask-sdk-controls-typedoc.s3-website-us-east-1.amazonaws.com)
-
-## Samples
-
-### [Hello World](https://github.com/alexa/skill-sample-controls-hello-world)
-
-Sample that familiarizes you with the Controls framework, allowing you to hear a response from Alexa when you trigger the skill.
-
-### [Fruit Shop](https://github.com/alexa/skill-sample-controls-fruit-shop)
-
-Build a multi-modal grocery shopping skill using custom and library controls for item lists, shopping cart management, and checkout.
-
-## Opening Issues
-
-For bug reports, feature requests and questions, we would like to hear about it. Search the [existing issues](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/issues) and try to make sure your problem doesn’t already exist before opening a new issue. It’s helpful if you include the version of the SDK, Node.js or browser environment and OS you’re using. Please include a stack trace and reduced repro case when appropriate, too.
-
-## License
-
-This SDK is distributed under the Apache License, Version 2.0, see LICENSE for more information.
-
-## Got Feedback?
-
-Request and vote for Alexa features [here](https://alexa.uservoice.com/forums/906892-alexa-skills-developer-voice-and-vote/filters/top?category_id=322783)!
+You can find the Github Repo for ASK SDK Control [here](https://github.com/alexa/ask-sdk-controls)
```

Comment:
hyperlinks with text like (here) isn't a great practice.  better is to have the text be the name of the page or the URL.

e.g.
```
The project has been moved to [ASK SDK Controls Github](https://github.com/alexa/ask-sdk-controls)
```
or
```
You can find the Github Repo for ASK SDK Control at [https://github.com/alexa/ask-sdk-controls](https://github.com/alexa/ask-sdk-controls)
```

Reply:
Sure, updated in latest revision!